content/news/doomhammer-tournament-announcement.md

```diff
@@ -0,0 +1,20 @@
+<div style="text-align:center">
+<iframe width="600" height="333" src="//www.youtube.com/embed/o9XJ-vqoh98?feature=player_detailpage" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Greetings ladies and gentlemen,
+
+our good friend Hecthor Doomhammer is eager to welcome you all to the first ever Hecthor Doomhammer Red Alert 1vs1 Tournament!
```

Comment:
Minor nit: it's generally good practice to avoid line-breaking in the middle of a title like Red Alert.